README.md

```diff
@@ -113,7 +113,27 @@ h := widget.NewHexWidget()
 h.Set(0xf)
 ```
 
-## Validation
+## Data Binding
+
+Community contributed data sources for binding.
+
+`import fyne.io/x/fyne/data/binding`
+
+### WebString
+
+A `WebSocketString` binding creates a data binding of type `String` to the specified web socket URL.
```

Comment:
Seems a little odd as `String` is the data binding which binds to a `string`.

```suggestion
A `WebSocketString` binding creates a `String` binding to the specified web socket URL.
```

or

```suggestion
A `WebSocketString` binding creates a data binding of type `string` to the specified web socket URL.
```

Reply:
Thanks. Good suggestions but I added more to remove ambiguity, I think the new doc is much clearer.